src/main/java/com/kodilla/ecommercee/domains/Group.java

```diff
@@ -3,33 +3,50 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 import javax.persistence.*;
 import javax.validation.constraints.NotNull;
 import java.util.ArrayList;
 import java.util.List;
 
-@NoArgsConstructor
-@AllArgsConstructor
+@Setter
 @Entity
-@Getter
 @Table(name = "GROUPS")
 public class Group {
 
+    private Long id;
+    private String groupName;
+
+    private List<Product> products = new ArrayList<>();
+
+    public Group (){
+    }
+
+    public Group(String groupName) {
+        this.groupName = groupName;
+    }
+
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue
     @NotNull
     @Column(name = "GROUP_ID", unique = true)
-    private Long groupId;
+    public Long getId(){
```

Comment:
Myślę  że jeżeli używamy loomboka, to lepiej zostać przy adnotacjach @Getter

Reply:
ok, zmiany wprowadzone